lib/build.js

```diff
@@ -408,6 +418,20 @@ export default {
           }
           GoogleAnalytics.sendEvent('build', 'failed');
         }
+        for (const error of this.errorMatcher.currentMatch) {
+          this.linter.setMessages([{
+            type: error.type || 'Error',
+            name: targetName,
+            text: error.message,
+            filePath: error.file,
+            range: [
+              [error.lineStart - 1 || error.line - 1,
+                error.columnStart - 1 || error.column - 1],
+              [error.lineEnd - 1 || error.line - 1,
+                error.columnEnd - 1 || error.column - 1]
```

Comment:
I'm attempting to use both the standard `build` named captures, and `linter` named captures.